Cargo.toml

```
@@ -162,6 +162,9 @@ harness = false
 name = "send_events"
 harness = false
 
+[workspace.dependencies]
+tokio = { version = "1.37.0" }
```

Thanks to @Hocuri for the idea

---

You probably also want to include the features here, otherwise running `cargo check` in the `deltachat-time` directory will fail.

(And it looks like we forgot to specify tokio's `sync` feature, which we didn't notice because some other dependency enabled it for us)

```suggestion
tokio = { version = "1.37.0", features = ["fs", "rt-multi-thread", "macros", "sync"] }
```